{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "91fb6ec7",
   "metadata": {},
   "source": [
    "# 프로젝트 개요\n",
    "\n",
    "유전자 발현 데이터를 활용한 질병 관련 유전자 탐색\n",
    "\n",
    "본 프로젝트는 GSE65858 유전체 발현 데이터셋을 활용, **두경부암**에 걸린 세포와 건강한 세포 사이에 유젼자들의 발현량에 어떤 차이가 있는지 알아보는 것을 목표로 함.\n",
    "생물 데이터의 특성을 이해하고, 통계와 코딩으로 분석하여 의미있는 결과를 찾아낼 것.\n",
    "데이터는 NCBI(미국 국립생물정보센터)의 GEO라는 공공 데이터베이스에서 가져옴.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5cee867f",
   "metadata": {},
   "source": [
    "# 데이터셋 정보 확인 및 문제 정의\n",
    "\n",
    "## 문제 정의: 파일 불러오기 실패\n",
    "\n",
    "`pandas` 라이브러리의 `read_csv` 함수를 사용하여 파일 불러오는 과정에서 오류 직면. 일반적으로 CSV 파일은 첫 번째 행에 열 이름이 있고, 그 아래부터 데이터가 시작되므로 특별한 처리 없이 바로 불러올 수 있다고 생각함.\n",
    "`df=pd.read_csv(file_path)` 코드를 실행한 결과, `ParseError` 발생."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2d1ebcfd",
   "metadata": {},
   "outputs": [
    {
     "ename": "ParserError",
     "evalue": "Error tokenizing data. C error: Expected 3 fields in line 7, saw 8\n",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mParserError\u001b[39m                               Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[4]\u001b[39m\u001b[32m, line 6\u001b[39m\n\u001b[32m      4\u001b[39m \u001b[38;5;66;03m# 파일 경로를 지정합니다.\u001b[39;00m\n\u001b[32m      5\u001b[39m file_path = os.path.join(\u001b[33m\"\u001b[39m\u001b[33m..\u001b[39m\u001b[33m\"\u001b[39m, \u001b[33m\"\u001b[39m\u001b[33mdata\u001b[39m\u001b[33m\"\u001b[39m, \u001b[33m\"\u001b[39m\u001b[33mGSE65858_series_matrix.txt\u001b[39m\u001b[33m\"\u001b[39m)\n\u001b[32m----> \u001b[39m\u001b[32m6\u001b[39m df= \u001b[43mpd\u001b[49m\u001b[43m.\u001b[49m\u001b[43mread_csv\u001b[49m\u001b[43m(\u001b[49m\u001b[43mfile_path\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m      7\u001b[39m df.info()\n",
      "\u001b[36mFile \u001b[39m\u001b[32mc:\\Users\\dbsgm\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pandas\\io\\parsers\\readers.py:1026\u001b[39m, in \u001b[36mread_csv\u001b[39m\u001b[34m(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, date_format, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, encoding_errors, dialect, on_bad_lines, delim_whitespace, low_memory, memory_map, float_precision, storage_options, dtype_backend)\u001b[39m\n\u001b[32m   1013\u001b[39m kwds_defaults = _refine_defaults_read(\n\u001b[32m   1014\u001b[39m     dialect,\n\u001b[32m   1015\u001b[39m     delimiter,\n\u001b[32m   (...)\u001b[39m\u001b[32m   1022\u001b[39m     dtype_backend=dtype_backend,\n\u001b[32m   1023\u001b[39m )\n\u001b[32m   1024\u001b[39m kwds.update(kwds_defaults)\n\u001b[32m-> \u001b[39m\u001b[32m1026\u001b[39m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_read\u001b[49m\u001b[43m(\u001b[49m\u001b[43mfilepath_or_buffer\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mkwds\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[36mFile \u001b[39m\u001b[32mc:\\Users\\dbsgm\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pandas\\io\\parsers\\readers.py:626\u001b[39m, in \u001b[36m_read\u001b[39m\u001b[34m(filepath_or_buffer, kwds)\u001b[39m\n\u001b[32m    623\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m parser\n\u001b[32m    625\u001b[39m \u001b[38;5;28;01mwith\u001b[39;00m parser:\n\u001b[32m--> \u001b[39m\u001b[32m626\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mparser\u001b[49m\u001b[43m.\u001b[49m\u001b[43mread\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnrows\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[36mFile \u001b[39m\u001b[32mc:\\Users\\dbsgm\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pandas\\io\\parsers\\readers.py:1923\u001b[39m, in \u001b[36mTextFileReader.read\u001b[39m\u001b[34m(self, nrows)\u001b[39m\n\u001b[32m   1916\u001b[39m nrows = validate_integer(\u001b[33m\"\u001b[39m\u001b[33mnrows\u001b[39m\u001b[33m\"\u001b[39m, nrows)\n\u001b[32m   1917\u001b[39m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m   1918\u001b[39m     \u001b[38;5;66;03m# error: \"ParserBase\" has no attribute \"read\"\u001b[39;00m\n\u001b[32m   1919\u001b[39m     (\n\u001b[32m   1920\u001b[39m         index,\n\u001b[32m   1921\u001b[39m         columns,\n\u001b[32m   1922\u001b[39m         col_dict,\n\u001b[32m-> \u001b[39m\u001b[32m1923\u001b[39m     ) = \u001b[38;5;28;43mself\u001b[39;49m\u001b[43m.\u001b[49m\u001b[43m_engine\u001b[49m\u001b[43m.\u001b[49m\u001b[43mread\u001b[49m\u001b[43m(\u001b[49m\u001b[43m  \u001b[49m\u001b[38;5;66;43;03m# type: ignore[attr-defined]\u001b[39;49;00m\n\u001b[32m   1924\u001b[39m \u001b[43m        \u001b[49m\u001b[43mnrows\u001b[49m\n\u001b[32m   1925\u001b[39m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m   1926\u001b[39m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m:\n\u001b[32m   1927\u001b[39m     \u001b[38;5;28mself\u001b[39m.close()\n",
      "\u001b[36mFile \u001b[39m\u001b[32mc:\\Users\\dbsgm\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pandas\\io\\parsers\\c_parser_wrapper.py:234\u001b[39m, in \u001b[36mCParserWrapper.read\u001b[39m\u001b[34m(self, nrows)\u001b[39m\n\u001b[32m    232\u001b[39m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m    233\u001b[39m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mself\u001b[39m.low_memory:\n\u001b[32m--> \u001b[39m\u001b[32m234\u001b[39m         chunks = \u001b[38;5;28;43mself\u001b[39;49m\u001b[43m.\u001b[49m\u001b[43m_reader\u001b[49m\u001b[43m.\u001b[49m\u001b[43mread_low_memory\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnrows\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    235\u001b[39m         \u001b[38;5;66;03m# destructive to chunks\u001b[39;00m\n\u001b[32m    236\u001b[39m         data = _concatenate_chunks(chunks)\n",
      "\u001b[36mFile \u001b[39m\u001b[32mpandas/_libs/parsers.pyx:838\u001b[39m, in \u001b[36mpandas._libs.parsers.TextReader.read_low_memory\u001b[39m\u001b[34m()\u001b[39m\n",
      "\u001b[36mFile \u001b[39m\u001b[32mpandas/_libs/parsers.pyx:905\u001b[39m, in \u001b[36mpandas._libs.parsers.TextReader._read_rows\u001b[39m\u001b[34m()\u001b[39m\n",
      "\u001b[36mFile \u001b[39m\u001b[32mpandas/_libs/parsers.pyx:874\u001b[39m, in \u001b[36mpandas._libs.parsers.TextReader._tokenize_rows\u001b[39m\u001b[34m()\u001b[39m\n",
      "\u001b[36mFile \u001b[39m\u001b[32mpandas/_libs/parsers.pyx:891\u001b[39m, in \u001b[36mpandas._libs.parsers.TextReader._check_tokenize_status\u001b[39m\u001b[34m()\u001b[39m\n",
      "\u001b[36mFile \u001b[39m\u001b[32mpandas/_libs/parsers.pyx:2061\u001b[39m, in \u001b[36mpandas._libs.parsers.raise_parser_error\u001b[39m\u001b[34m()\u001b[39m\n",
      "\u001b[31mParserError\u001b[39m: Error tokenizing data. C error: Expected 3 fields in line 7, saw 8\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "\n",
    "# 파일 경로를 지정합니다.\n",
    "file_path = os.path.join(\"..\", \"data\", \"GSE65858_series_matrix.txt\")\n",
    "df= pd.read_csv(file_path)\n",
    "df.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "259dec75",
   "metadata": {},
   "source": [
    "### Raw 데이터 분석(텍스트 파일 형태로 열어 확인)\n",
    "`ParseError`의 원인을 파악하기 위해 파일을 텍스트 모드로 열어 확인. 다음코드를 이용하여 상위 20줄 출력."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b1a9458d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "!Series_title\t\"Gene expression patterns and TP53 mutations are associated with HPV RNA status, lymph node metastasis, and survival in head and neck cancer\"\n",
      "!Series_geo_accession\t\"GSE65858\"\n",
      "!Series_status\t\"Public on Jun 24 2015\"\n",
      "!Series_submission_date\t\"Feb 11 2015\"\n",
      "!Series_last_update_date\t\"Aug 13 2018\"\n",
      "!Series_pubmed_id\t\"26095926\"\n",
      "!Series_summary\t\"A growing proportion of head and neck squamous cell carcinomas (HNSCC) is associated with the human papilloma virus (HPV), particularly HPV16. We compare tumors with different HPV16 DNA and RNA (E6*I) status from 290 consecutively recruited HNSCC patients by gene expression profiling and targeted sequencing of 50 genes. We confirm that the HPV16 DNA+ RNA+ tumors are molecularly distinct from the HPV-negative (DNA-) HNSCC and have elevated expression of cell cycle genes and rare TP53 mutations (3.6%, 1/28). We show that tumors with non-transcriptionally active HPV16 (DNA+ RNA-) are similar to HPV DNA- tumors regarding gene expression and frequency of TP53 mutations (47%, 8/17 and 43%, 72/167, respectively). Furthermore, we identify four gene expression clusters. They moderately but significantly differ in overall survival. One cluster exhibits high expression of immune response genes (IR) and contains most of the HPV16 DNA+ RNA+ patients. The IR cluster and disruptive TP53 mutations are associated with lymph node metastasis independent of HPV16 status and we validate each of these associations in another large data set. Consistent with earlier studies, disruptive TP53 mutations are prognostically unfavorable. Our findings underscore the importance of measuring the HPV16 RNA (E6*I) and TP53-mutation status for patient stratification and for the first time identify associations of an immune response-related gene expression cluster and TP53 mutations with lymph node metastasis in HNSCC.\"\n",
      "!Series_overall_design\t\"A total of 300 samples were considered. Quality control procedures were applied to microarray probe-level intensity files. A total of 270 tumor arrays remained after removing low-quality arrays, duplicate arrays, and arrays from non-HNSCC samples. Expression values were log2-transformed and normalized using RSN.\"\n",
      "!Series_type\t\"Expression profiling by array\"\n",
      "!Series_contributor\t\"Gunnar,,Wichmann\"\n",
      "!Series_contributor\t\"Maciej,,Rosolowski\"\n",
      "!Series_sample_id\t\"GSM1607684 GSM1607685 GSM1607686 GSM1607687 GSM1607688 GSM1607689 GSM1607690 GSM1607691 GSM1607692 GSM1607693 GSM1607694 GSM1607695 GSM1607696 GSM1607697 GSM1607698 GSM1607699 GSM1607700 GSM1607701 GSM1607702 GSM1607703 GSM1607704 GSM1607705 GSM1607706 GSM1607707 GSM1607708 GSM1607709 GSM1607710 GSM1607711 GSM1607712 GSM1607713 GSM1607714 GSM1607715 GSM1607716 GSM1607717 GSM1607718 GSM1607719 GSM1607720 GSM1607721 GSM1607722 GSM1607723 GSM1607724 GSM1607725 GSM1607726 GSM1607727 GSM1607728 GSM1607729 GSM1607730 GSM1607731 GSM1607732 GSM1607733 GSM1607734 GSM1607735 GSM1607736 GSM1607737 GSM1607738 GSM1607739 GSM1607740 GSM1607741 GSM1607742 GSM1607743 GSM1607744 GSM1607745 GSM1607746 GSM1607747 GSM1607748 GSM1607749 GSM1607750 GSM1607751 GSM1607752 GSM1607753 GSM1607754 GSM1607755 GSM1607756 GSM1607757 GSM1607758 GSM1607759 GSM1607760 GSM1607761 GSM1607762 GSM1607763 GSM1607764 GSM1607765 GSM1607766 GSM1607767 GSM1607768 GSM1607769 GSM1607770 GSM1607771 GSM1607772 GSM1607773 GSM1607774 GSM1607775 GSM1607776 GSM1607777 GSM1607778 GSM1607779 GSM1607780 GSM1607781 GSM1607782 GSM1607783 GSM1607784 GSM1607785 GSM1607786 GSM1607787 GSM1607788 GSM1607789 GSM1607790 GSM1607791 GSM1607792 GSM1607793 GSM1607794 GSM1607795 GSM1607796 GSM1607797 GSM1607798 GSM1607799 GSM1607800 GSM1607801 GSM1607802 GSM1607803 GSM1607804 GSM1607805 GSM1607806 GSM1607807 GSM1607808 GSM1607809 GSM1607810 GSM1607811 GSM1607812 GSM1607813 GSM1607814 GSM1607815 GSM1607816 GSM1607817 GSM1607818 GSM1607819 GSM1607820 GSM1607821 GSM1607822 GSM1607823 GSM1607824 GSM1607825 GSM1607826 GSM1607827 GSM1607828 GSM1607829 GSM1607830 GSM1607831 GSM1607832 GSM1607833 GSM1607834 GSM1607835 GSM1607836 GSM1607837 GSM1607838 GSM1607839 GSM1607840 GSM1607841 GSM1607842 GSM1607843 GSM1607844 GSM1607845 GSM1607846 GSM1607847 GSM1607848 GSM1607849 GSM1607850 GSM1607851 GSM1607852 GSM1607853 GSM1607854 GSM1607855 GSM1607856 GSM1607857 GSM1607858 GSM1607859 GSM1607860 GSM1607861 GSM1607862 GSM1607863 GSM1607864 GSM1607865 GSM1607866 GSM1607867 GSM1607868 GSM1607869 GSM1607870 GSM1607871 GSM1607872 GSM1607873 GSM1607874 GSM1607875 GSM1607876 GSM1607877 GSM1607878 GSM1607879 GSM1607880 GSM1607881 GSM1607882 GSM1607883 GSM1607884 GSM1607885 GSM1607886 GSM1607887 GSM1607888 GSM1607889 GSM1607890 GSM1607891 GSM1607892 GSM1607893 GSM1607894 GSM1607895 GSM1607896 GSM1607897 GSM1607898 GSM1607899 GSM1607900 GSM1607901 GSM1607902 GSM1607903 GSM1607904 GSM1607905 GSM1607906 GSM1607907 GSM1607908 GSM1607909 GSM1607910 GSM1607911 GSM1607912 GSM1607913 GSM1607914 GSM1607915 GSM1607916 GSM1607917 GSM1607918 GSM1607919 GSM1607920 GSM1607921 GSM1607922 GSM1607923 GSM1607924 GSM1607925 GSM1607926 GSM1607927 GSM1607928 GSM1607929 GSM1607930 GSM1607931 GSM1607932 GSM1607933 GSM1607934 GSM1607935 GSM1607936 GSM1607937 GSM1607938 GSM1607939 GSM1607940 GSM1607941 GSM1607942 GSM1607943 GSM1607944 GSM1607945 GSM1607946 GSM1607947 GSM1607948 GSM1607949 GSM1607950 GSM1607951 GSM1607952 GSM1607953 \"\n",
      "!Series_contact_name\t\"Maciej,,Rosolowski\"\n",
      "!Series_contact_email\t\"maciej.rosolowski@imise.uni-leipzig.de\"\n",
      "!Series_contact_institute\t\"Institute for Medical Informatics, Statistics and Epidemiology (IMISE)\"\n",
      "!Series_contact_address\t\"Härtelstr. 16-18\"\n",
      "!Series_contact_city\t\"Leipzig\"\n",
      "!Series_contact_zip/postal_code\t\"04107\"\n",
      "!Series_contact_country\t\"Germany\"\n",
      "!Series_supplementary_file\t\"ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE65nnn/GSE65858/suppl/GSE65858_Non-normalized_data.txt.gz\"\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "\n",
    "file_path = os.path.join(\"..\", \"data\", \"GSE65858_series_matrix.txt\")\n",
    "\n",
    "with open(file_path, 'r', encoding='utf-8') as f:\n",
    "    for i in range(20):\n",
    "        print(f.readline(), end='')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e6e61e7",
   "metadata": {},
   "source": [
    "### 오류 원인 분석\n",
    "`ParseError`는 파일의 구조가 Pandas가 예상하는 CSV형식과 다르다는 것을 의미. 직접 파일을 열어 확인한 결과, 파일 상단에 `#`와 `!`로 시작하는 많은 부가 정보가 존재. Pandas는 이 부가 정보를 데이터의 일부로 잘못 인식하여 오류가 발생함.\n",
    "\n",
    "\n",
    "`ParserError: Error tokenizing data. C error: Expected 3 fields in line 7, saw 8` : 7번째 줄에 데이터가 3개의 필드로 나뉘어야 하는데 8개의 필드가 발견\n",
    "\n",
    "따라서 이 부가 정보를 건너뛰고, 실제 데이터가 시작되는 행부터 파일을 읽어와야 한다고 판단함."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "aea7b3e9",
   "metadata": {},
   "source": [
    "### 해결 과정\n",
    "\n",
    "`!`로 시작하는 메타데이터를 모두 건너뛰고, 실제 데이터가 시작되는 지점부터 파일을 읽도록 코드를 수정."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "f1750de0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\"ID_REF\"\t\"GSM1607684\"\t\"GSM1607685\"\t\"GSM1607686\"\t\"GSM1607687\"\t\"GSM1607688\"\t\"GSM1607689\"\t\"GSM1607690\"\t\"GSM1607691\"\t\"GSM1607692\"\t\"GSM1607693\"\t\"GSM1607694\"\t\"GSM1607695\"\t\"GSM1607696\"\t\"GSM1607697\"\t\"GSM1607698\"\t\"GSM1607699\"\t\"GSM1607700\"\t\"GSM1607701\"\t\"GSM1607702\"\t\"GSM1607703\"\t\"GSM1607704\"\t\"GSM1607705\"\t\"GSM1607706\"\t\"GSM1607707\"\t\"GSM1607708\"\t\"GSM1607709\"\t\"GSM1607710\"\t\"GSM1607711\"\t\"GSM1607712\"\t\"GSM1607713\"\t\"GSM1607714\"\t\"GSM1607715\"\t\"GSM1607716\"\t\"GSM1607717\"\t\"GSM1607718\"\t\"GSM1607719\"\t\"GSM1607720\"\t\"GSM1607721\"\t\"GSM1607722\"\t\"GSM1607723\"\t\"GSM1607724\"\t\"GSM1607725\"\t\"GSM1607726\"\t\"GSM1607727\"\t\"GSM1607728\"\t\"GSM1607729\"\t\"GSM1607730\"\t\"GSM1607731\"\t\"GSM1607732\"\t\"GSM1607733\"\t\"GSM1607734\"\t\"GSM1607735\"\t\"GSM1607736\"\t\"GSM1607737\"\t\"GSM1607738\"\t\"GSM1607739\"\t\"GSM1607740\"\t\"GSM1607741\"\t\"GSM1607742\"\t\"GSM1607743\"\t\"GSM1607744\"\t\"GSM1607745\"\t\"GSM1607746\"\t\"GSM1607747\"\t\"GSM1607748\"\t\"GSM1607749\"\t\"GSM1607750\"\t\"GSM1607751\"\t\"GSM1607752\"\t\"GSM1607753\"\t\"GSM1607754\"\t\"GSM1607755\"\t\"GSM1607756\"\t\"GSM1607757\"\t\"GSM1607758\"\t\"GSM1607759\"\t\"GSM1607760\"\t\"GSM1607761\"\t\"GSM1607762\"\t\"GSM1607763\"\t\"GSM1607764\"\t\"GSM1607765\"\t\"GSM1607766\"\t\"GSM1607767\"\t\"GSM1607768\"\t\"GSM1607769\"\t\"GSM1607770\"\t\"GSM1607771\"\t\"GSM1607772\"\t\"GSM1607773\"\t\"GSM1607774\"\t\"GSM1607775\"\t\"GSM1607776\"\t\"GSM1607777\"\t\"GSM1607778\"\t\"GSM1607779\"\t\"GSM1607780\"\t\"GSM1607781\"\t\"GSM1607782\"\t\"GSM1607783\"\t\"GSM1607784\"\t\"GSM1607785\"\t\"GSM1607786\"\t\"GSM1607787\"\t\"GSM1607788\"\t\"GSM1607789\"\t\"GSM1607790\"\t\"GSM1607791\"\t\"GSM1607792\"\t\"GSM1607793\"\t\"GSM1607794\"\t\"GSM1607795\"\t\"GSM1607796\"\t\"GSM1607797\"\t\"GSM1607798\"\t\"GSM1607799\"\t\"GSM1607800\"\t\"GSM1607801\"\t\"GSM1607802\"\t\"GSM1607803\"\t\"GSM1607804\"\t\"GSM1607805\"\t\"GSM1607806\"\t\"GSM1607807\"\t\"GSM1607808\"\t\"GSM1607809\"\t\"GSM1607810\"\t\"GSM1607811\"\t\"GSM1607812\"\t\"GSM1607813\"\t\"GSM1607814\"\t\"GSM1607815\"\t\"GSM1607816\"\t\"GSM1607817\"\t\"GSM1607818\"\t\"GSM1607819\"\t\"GSM1607820\"\t\"GSM1607821\"\t\"GSM1607822\"\t\"GSM1607823\"\t\"GSM1607824\"\t\"GSM1607825\"\t\"GSM1607826\"\t\"GSM1607827\"\t\"GSM1607828\"\t\"GSM1607829\"\t\"GSM1607830\"\t\"GSM1607831\"\t\"GSM1607832\"\t\"GSM1607833\"\t\"GSM1607834\"\t\"GSM1607835\"\t\"GSM1607836\"\t\"GSM1607837\"\t\"GSM1607838\"\t\"GSM1607839\"\t\"GSM1607840\"\t\"GSM1607841\"\t\"GSM1607842\"\t\"GSM1607843\"\t\"GSM1607844\"\t\"GSM1607845\"\t\"GSM1607846\"\t\"GSM1607847\"\t\"GSM1607848\"\t\"GSM1607849\"\t\"GSM1607850\"\t\"GSM1607851\"\t\"GSM1607852\"\t\"GSM1607853\"\t\"GSM1607854\"\t\"GSM1607855\"\t\"GSM1607856\"\t\"GSM1607857\"\t\"GSM1607858\"\t\"GSM1607859\"\t\"GSM1607860\"\t\"GSM1607861\"\t\"GSM1607862\"\t\"GSM1607863\"\t\"GSM1607864\"\t\"GSM1607865\"\t\"GSM1607866\"\t\"GSM1607867\"\t\"GSM1607868\"\t\"GSM1607869\"\t\"GSM1607870\"\t\"GSM1607871\"\t\"GSM1607872\"\t\"GSM1607873\"\t\"GSM1607874\"\t\"GSM1607875\"\t\"GSM1607876\"\t\"GSM1607877\"\t\"GSM1607878\"\t\"GSM1607879\"\t\"GSM1607880\"\t\"GSM1607881\"\t\"GSM1607882\"\t\"GSM1607883\"\t\"GSM1607884\"\t\"GSM1607885\"\t\"GSM1607886\"\t\"GSM1607887\"\t\"GSM1607888\"\t\"GSM1607889\"\t\"GSM1607890\"\t\"GSM1607891\"\t\"GSM1607892\"\t\"GSM1607893\"\t\"GSM1607894\"\t\"GSM1607895\"\t\"GSM1607896\"\t\"GSM1607897\"\t\"GSM1607898\"\t\"GSM1607899\"\t\"GSM1607900\"\t\"GSM1607901\"\t\"GSM1607902\"\t\"GSM1607903\"\t\"GSM1607904\"\t\"GSM1607905\"\t\"GSM1607906\"\t\"GSM1607907\"\t\"GSM1607908\"\t\"GSM1607909\"\t\"GSM1607910\"\t\"GSM1607911\"\t\"GSM1607912\"\t\"GSM1607913\"\t\"GSM1607914\"\t\"GSM1607915\"\t\"GSM1607916\"\t\"GSM1607917\"\t\"GSM1607918\"\t\"GSM1607919\"\t\"GSM1607920\"\t\"GSM1607921\"\t\"GSM1607922\"\t\"GSM1607923\"\t\"GSM1607924\"\t\"GSM1607925\"\t\"GSM1607926\"\t\"GSM1607927\"\t\"GSM1607928\"\t\"GSM1607929\"\t\"GSM1607930\"\t\"GSM1607931\"\t\"GSM1607932\"\t\"GSM1607933\"\t\"GSM1607934\"\t\"GSM1607935\"\t\"GSM1607936\"\t\"GSM1607937\"\t\"GSM1607938\"\t\"GSM1607939\"\t\"GSM1607940\"\t\"GSM1607941\"\t\"GSM1607942\"\t\"GSM1607943\"\t\"GSM1607944\"\t\"GSM1607945\"\t\"GSM1607946\"\t\"GSM1607947\"\t\"GSM1607948\"\t\"GSM1607949\"\t\"GSM1607950\"\t\"GSM1607951\"\t\"GSM1607952\"\t\"GSM1607953\"\n",
      "\"ILMN_1343291\"\t13.1886\t13.4787\t13.2263\t13.44\t13.2678\t13.4199\t13.3665\t13.7561\t13.3718\t13.3454\t13.3311\t13.4233\t13.5914\t13.444\t13.3687\t13.4681\t13.0214\t13.3123\t13.4475\t13.1651\t13.4636\t13.5069\t13.4979\t13.6276\t13.3919\t13.4546\t13.2629\t13.5722\t13.3501\t12.9068\t13.3584\t13.3846\t13.4179\t13.0966\t13.4603\t13.628\t13.4234\t13.3191\t13.6203\t13.5211\t13.4753\t13.4769\t13.1749\t13.2887\t13.3154\t13.3675\t13.2925\t13.3201\t13.4009\t13.3613\t13.4258\t13.5002\t12.9048\t13.4262\t13.4386\t13.3476\t13.4695\t13.3135\t13.0077\t13.3378\t13.4741\t13.4768\t13.4274\t13.422\t13.5744\t13.3451\t13.4465\t13.4583\t13.4829\t13.2625\t13.2203\t13.3042\t13.3847\t13.4884\t13.3857\t13.5604\t13.2488\t13.53\t13.6677\t13.4255\t13.3869\t13.251\t13.4594\t13.4341\t12.9564\t13.4633\t13.3094\t13.4205\t13.2852\t13.3053\t13.4242\t13.5484\t13.0679\t13.4655\t13.4335\t13.5224\t13.1311\t13.454\t13.4774\t13.4325\t13.4994\t13.6219\t13.5741\t13.3882\t13.5796\t13.43\t13.4231\t13.5025\t13.6193\t13.377\t13.3288\t13.1189\t13.3206\t13.487\t13.3286\t13.3908\t13.0764\t13.4046\t13.5172\t13.3901\t13.6828\t13.5428\t13.2549\t13.3331\t13.4163\t13.1769\t13.4193\t13.3408\t13.4654\t13.0897\t13.5131\t13.477\t13.5076\t13.2505\t13.46\t13.5617\t13.6425\t13.198\t13.3384\t13.5923\t13.548\t13.5951\t13.4995\t13.3804\t13.263\t13.159\t13.2947\t13.1693\t13.5031\t13.4019\t13.3152\t13.4744\t13.3382\t13.5055\t13.5709\t13.7153\t13.198\t13.4581\t13.5188\t13.487\t13.4674\t13.6858\t13.11\t13.48\t13.5384\t13.5197\t13.3975\t13.3954\t13.1873\t13.4706\t13.6051\t13.4549\t13.3924\t13.169\t13.3701\t13.5177\t13.5046\t13.4231\t13.1525\t13.4811\t13.421\t13.2935\t13.428\t13.5455\t13.59\t13.3166\t13.4577\t13.6573\t13.3864\t13.3267\t13.2623\t13.3853\t13.604\t13.5152\t13.5234\t13.4462\t13.349\t13.2843\t13.2916\t13.4529\t13.6927\t13.4689\t13.4306\t13.6804\t13.3299\t13.479\t13.3706\t13.1812\t13.53\t13.4692\t13.468\t13.5397\t13.2786\t13.3023\t13.4968\t13.2622\t13.3227\t13.7193\t13.4742\t13.5589\t13.4055\t13.506\t13.5398\t13.3549\t13.2396\t13.4617\t13.3534\t13.5297\t13.5155\t13.3903\t13.4539\t13.4251\t13.244\t13.4893\t13.2968\t13.5462\t13.3813\t13.3217\t13.4159\t13.1199\t13.3462\t13.3604\t13.2447\t13.3246\t13.5851\t13.3773\t13.4136\t13.5885\t13.4487\t13.2241\t13.3534\t13.2357\t13.2975\t13.6188\t13.4299\t13.4533\t13.5638\t13.589\t13.4859\t13.2922\t13.5946\t13.3038\t13.4352\t13.1511\t13.5872\t13.4713\t13.5859\t13.4941\t13.483\t13.4524\n",
      "\"ILMN_1343295\"\t11.5546\t11.1959\t11.0929\t11.7418\t10.4042\t11.7005\t11.1597\t11.4082\t11.9006\t11.0817\t10.6383\t10.8353\t10.6179\t12.065\t11.7824\t11.3006\t11.7502\t10.7156\t11.3043\t11.4232\t11.632\t11.7664\t11.1929\t11.99\t12.2412\t11.7199\t11.633\t11.0829\t11.3338\t11.995\t11.8657\t11.1298\t11.28\t11.1548\t10.4223\t11.0186\t11.1511\t11.4986\t11.8043\t12.2931\t11.2507\t11.9215\t10.8253\t11.4012\t11.4805\t10.5848\t12.2329\t11.1913\t11.6733\t11.4715\t11.2572\t11.1642\t11.3408\t10.8626\t11.1305\t11.3786\t11.1634\t11.6765\t12.1206\t11.483\t11.1437\t11.1046\t11.8796\t11.0322\t11.495\t11.1015\t10.9902\t10.8158\t11.5153\t10.7686\t11.2074\t11.4286\t11.3251\t10.9981\t10.7694\t11.3334\t11.3724\t10.8099\t11.2283\t12.0695\t12.3284\t11.3081\t11.1329\t11.283\t11.8784\t11.1538\t11.2358\t11.0944\t11.2269\t12.0175\t11.3057\t11.3206\t10.7812\t11.7552\t11.4088\t10.5731\t11.7247\t12.139\t11.4913\t11.1082\t11.571\t11.2743\t11.8763\t11.5387\t11.4744\t11.03\t12.3398\t11.1439\t10.965\t10.8662\t11.2237\t11.0728\t11.9769\t11.1444\t11.4262\t11.1679\t11.813\t11.2998\t11.9723\t12.1187\t11.0974\t10.7446\t11.2997\t11.3567\t11.2519\t11.7048\t10.8741\t11.6519\t11.7715\t11.7967\t10.9497\t11.1726\t11.492\t11.3018\t11.2136\t11.4617\t11.4622\t11.4924\t11.3523\t10.7334\t11.658\t11.528\t10.8658\t12.0959\t11.2238\t11.3964\t12.1304\t11.2549\t11.5312\t11.6846\t10.9471\t12.0243\t11.9115\t11.3974\t11.0423\t11.0414\t11.0845\t11.2255\t11.5049\t10.9568\t10.5254\t11.4787\t11.3129\t11.3015\t12.2229\t11.5478\t11.2354\t11.6968\t11.4787\t10.8363\t11.2315\t11.3721\t11.297\t12.2523\t11.9589\t11.0896\t11.9947\t11.1744\t12.0941\t10.9109\t11.6121\t12.0076\t11.3125\t10.9628\t11.4035\t11.0048\t11.4505\t11.4248\t11.8891\t11.7309\t10.5838\t11.6736\t11.4983\t11.1704\t10.8191\t11.58\t11.4517\t11.7684\t11.6904\t11.9692\t11.2469\t11.4846\t11.7208\t10.6526\t11.1971\t11.3134\t11.1621\t10.9053\t11.1724\t11.6833\t11.7411\t12.4053\t11.299\t11.5819\t11.1125\t11.6436\t11.1245\t11.156\t10.9708\t11.7193\t11.3559\t10.9945\t11.7416\t11.7625\t11.4481\t10.7157\t11.5543\t11.7965\t10.8809\t10.6325\t11.0499\t10.8976\t11.251\t11.5327\t10.142\t11.5446\t11.6663\t11.6031\t11.0521\t11.2588\t11.4583\t12.0598\t10.9866\t10.9663\t11.5236\t10.7011\t11.1929\t11.7359\t11.2627\t10.6793\t11.4298\t10.4843\t11.5536\t12.3349\t11.4133\t11.0134\t11.3429\t11.0176\t11.5474\t10.5752\t10.9917\t11.4681\t11.6038\t10.2602\t11.7133\t11.6153\t10.6864\t11.178\t11.6972\t11.0559\n",
      "\"ILMN_1651209\"\t6.4138\t6.4093\t6.368\t6.3737\t6.4413\t6.4673\t6.5391\t6.3631\t6.4349\t6.3864\t6.5408\t6.4198\t6.2816\t6.4778\t6.4341\t6.3757\t6.4072\t6.4013\t6.467\t6.444\t6.5012\t6.37\t6.5208\t6.5304\t6.3765\t6.3886\t6.3791\t6.5494\t6.3948\t6.5328\t6.4655\t6.4144\t6.4918\t6.4222\t6.3591\t6.4889\t6.3486\t6.3433\t6.4642\t6.3859\t6.3738\t6.4266\t6.4121\t6.4322\t6.3784\t6.5175\t6.5046\t6.3555\t6.3446\t6.3953\t6.431\t6.4092\t6.4306\t6.5085\t6.3822\t6.3997\t6.4755\t6.3526\t6.4411\t6.4366\t6.4796\t6.478\t6.3532\t6.3327\t6.4593\t6.4026\t6.517\t6.3991\t6.3605\t6.4405\t6.5358\t6.2788\t6.3907\t6.3877\t6.4201\t6.4476\t6.4769\t6.4926\t6.4154\t6.3778\t6.3569\t6.3461\t6.3947\t6.3788\t6.631\t6.4104\t6.3646\t6.4375\t6.3955\t6.3547\t6.478\t6.3719\t6.3005\t6.4799\t6.392\t6.5069\t6.4493\t6.4171\t6.4472\t6.4506\t6.4593\t6.4386\t6.3152\t6.4025\t6.4279\t6.3372\t6.4091\t6.4222\t6.5179\t6.4042\t6.4898\t6.2764\t6.3365\t6.4868\t6.4958\t6.3849\t6.3713\t6.4124\t6.442\t6.2943\t6.337\t6.5185\t6.4994\t6.4123\t6.4803\t6.5576\t6.4314\t6.3626\t6.4165\t6.3747\t6.3858\t6.4196\t6.3537\t6.4602\t6.48\t6.3893\t6.3298\t6.4267\t6.373\t6.5797\t6.4821\t6.4086\t6.3276\t6.3804\t6.3396\t6.4235\t6.3551\t6.3915\t6.4383\t6.3782\t6.3825\t6.3765\t6.5239\t6.3696\t6.4803\t6.4394\t6.457\t6.2404\t6.5222\t6.4971\t6.5125\t6.3395\t6.425\t6.4678\t6.2977\t6.3612\t6.3778\t6.4231\t6.4078\t6.4395\t6.3576\t6.3796\t6.3649\t6.4417\t6.39\t6.4555\t6.388\t6.5323\t6.4595\t6.3744\t6.3816\t6.3505\t6.4057\t6.3919\t6.4041\t6.5773\t6.3833\t6.3951\t6.4191\t6.4121\t6.4384\t6.3505\t6.4013\t6.5159\t6.4472\t6.5613\t6.4299\t6.3994\t6.4201\t6.4159\t6.44\t6.3201\t6.4405\t6.3269\t6.4346\t6.3329\t6.5563\t6.4092\t6.4452\t6.3746\t6.3951\t6.3395\t6.5686\t6.405\t6.5309\t6.3627\t6.3892\t6.5013\t6.4755\t6.4316\t6.3437\t6.4786\t6.3173\t6.3904\t6.4028\t6.3045\t6.3787\t6.3587\t6.4083\t6.4417\t6.4014\t6.5129\t6.4084\t6.4324\t6.4322\t6.4136\t6.3928\t6.3956\t6.3658\t6.4377\t6.3239\t6.3424\t6.4481\t6.4457\t6.3342\t6.508\t6.3198\t6.4891\t6.3684\t6.4315\t6.4322\t6.4252\t6.5091\t6.4027\t6.3862\t6.3185\t6.3448\t6.3347\t6.4803\t6.4856\t6.4887\t6.349\t6.5698\t6.4373\t6.4324\t6.3617\t6.3804\t6.4352\t6.4303\t6.4314\n",
      "!Series_overall_design\t\"A total of 300 samples were considered. Quality control procedures were applied to microarray probe-level intensity files. A total of 270 tumor arrays remained after removing low-quality arrays, duplicate arrays, and arrays from non-HNSCC samples. Expression values were log2-transformed and normalized using RSN.\"\n",
      "!Series_supplementary_file\t\"ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE65nnn/GSE65858/suppl/GSE65858_Non-normalized_data.txt.gz\"\n",
      "!Sample_data_processing\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\t\"Raw data of all 47,323 probes was extracted by Illumina GenomeStudio without additional background correction. Data was further processed within R / Bioconductor. Expression values were log2-transformed and normalized using RSN. Batch effects of expression BeadChips were corrected using ComBat. Two criteria were used to remove arrays of low quality. First, the number of detected features with P<0.01 on an array was required to be within 3 interquartile ranges from the median of all arrays. 9 samples did not pass this criterion. Second, the Mahalanobis distance of several quality characteristics was computed for each array. These quality characteristics included signals from perfect-match and miss-match control probes, control probes present at different concentrations, mean of negative control probes, mean of house-keeping genes, number of expressed genes, mean signal strength of biotin-control-probes, and Euclidean distances of expression values. Thereby, Euclidean distances of expression values of each sample were based on their distances to the center, which was defined as the average of all samples after removing 10% samples farthest away from the center. The Mahalanobis distances of 4 arrays were more than 3 interquartile ranges away from the median of all arrays and were therefore excluded. Overall, of the assayed 300 samples 13 samples were excluded for quality reasons. Features detected by Illumina GenomeStudio with P<0.05 in less than 1% of the samples were excluded resulting in 31,347 features. Additionally, 17 feature were removed as their expression levels were still found to be significantly associated after Bonferroni-correction with batches of BeadChips. Finally, the preprocessing resulted in 31,330 features.  There were 12 paired measurements in the data including 10 pairs of samples taken from the same patients at short intervals, 1 pair of a primary and relapse tumor and 1 pair of biopsies from a first and second relapse. To remove the duplicates, the array with more detected features at P<0.01 was selected from each pair. Among the remaining 275 samples, 3 were taken from lymph node metastases and 2 were cell lines and 270 (included here) were tumor samples.\"\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "\n",
    "file_path = os.path.join(\"..\", \"data\", \"GSE65858_series_matrix.txt\")\n",
    "\n",
    "with open(file_path, 'r', encoding='utf-8') as f:\n",
    "    data_lines = 0\n",
    "    for line in f:\n",
    "        if not line.startswith('!'):\n",
    "            print(line, end='')\n",
    "            data_lines += 1\n",
    "\n",
    "            if data_lines >=5:\n",
    "                break\n",
    "\n",
    "\n",
    "\n",
    "with open(file_path, 'r', encoding='utf-8') as f:\n",
    "    data_lines = 0\n",
    "    for line in f:\n",
    "        # if line.startswith('!Sample_title'):\n",
    "        if 'normal' in line:\n",
    "            print(line, end='')\n",
    "        \n",
    "            \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "a2bbc420",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "파일 경로: ..\\data\\GSE65858_series_matrix.txt\n",
      "\n",
      "!Sample_characteristics_ch1\t\"ID: GW_001\"\t\"ID: GW_003\"\t\"ID: GW_004\"\t\"ID: GW_005\"\t\"ID: GW_006\"\t\"ID: GW_007\"\t\"ID: GW_008\"\t\"ID: GW_009\"\t\"ID: GW_010\"\t\"ID: GW_011\"\t\"ID: GW_012\"\t\"ID: GW_013\"\t\"ID: GW_014\"\t\"ID: GW_015\"\t\"ID: GW_016\"\t\"ID: GW_017\"\t\"ID: GW_018\"\t\"ID: GW_020\"\t\"ID: GW_021\"\t\"ID: GW_022\"\t\"ID: GW_023\"\t\"ID: GW_025\"\t\"ID: GW_026\"\t\"ID: GW_027\"\t\"ID: GW_028\"\t\"ID: GW_029\"\t\"ID: GW_030\"\t\"ID: GW_031\"\t\"ID: GW_032\"\t\"ID: GW_033\"\t\"ID: GW_034\"\t\"ID: GW_035\"\t\"ID: GW_036\"\t\"ID: GW_037\"\t\"ID: GW_038\"\t\"ID: GW_039\"\t\"ID: GW_040\"\t\"ID: GW_041\"\t\"ID: GW_042\"\t\"ID: GW_043\"\t\"ID: GW_044\"\t\"ID: GW_045\"\t\"ID: GW_046\"\t\"ID: GW_047\"\t\"ID: GW_049\"\t\"ID: GW_051\"\t\"ID: GW_052\"\t\"ID: GW_053\"\t\"ID: GW_054\"\t\"ID: GW_056\"\t\"ID: GW_057\"\t\"ID: GW_058\"\t\"ID: GW_059\"\t\"ID: GW_060\"\t\"ID: GW_061\"\t\"ID: GW_062\"\t\"ID: GW_063\"\t\"ID: GW_064\"\t\"ID: GW_065\"\t\"ID: GW_066\"\t\"ID: GW_067\"\t\"ID: GW_068\"\t\"ID: GW_069\"\t\"ID: GW_070\"\t\"ID: GW_071\"\t\"ID: GW_072\"\t\"ID: GW_073\"\t\"ID: GW_074\"\t\"ID: GW_075\"\t\"ID: GW_076\"\t\"ID: GW_077\"\t\"ID: GW_078\"\t\"ID: GW_079\"\t\"ID: GW_080\"\t\"ID: GW_081\"\t\"ID: GW_082\"\t\"ID: GW_083\"\t\"ID: GW_084\"\t\"ID: GW_085\"\t\"ID: GW_086\"\t\"ID: GW_087\"\t\"ID: GW_088\"\t\"ID: GW_089\"\t\"ID: GW_091\"\t\"ID: GW_092\"\t\"ID: GW_093\"\t\"ID: GW_094\"\t\"ID: GW_095\"\t\"ID: GW_096\"\t\"ID: GW_097\"\t\"ID: GW_101\"\t\"ID: GW_102\"\t\"ID: GW_103\"\t\"ID: GW_104\"\t\"ID: GW_105\"\t\"ID: GW_106\"\t\"ID: GW_107\"\t\"ID: GW_108\"\t\"ID: GW_109\"\t\"ID: GW_110\"\t\"ID: GW_111\"\t\"ID: GW_112\"\t\"ID: GW_113\"\t\"ID: GW_115\"\t\"ID: GW_116\"\t\"ID: GW_117\"\t\"ID: GW_118\"\t\"ID: GW_119\"\t\"ID: GW_120\"\t\"ID: GW_122\"\t\"ID: GW_123\"\t\"ID: GW_124\"\t\"ID: GW_125\"\t\"ID: GW_126\"\t\"ID: GW_127\"\t\"ID: GW_128\"\t\"ID: GW_129\"\t\"ID: GW_130\"\t\"ID: GW_131\"\t\"ID: GW_133\"\t\"ID: GW_134\"\t\"ID: GW_135\"\t\"ID: GW_136\"\t\"ID: GW_137\"\t\"ID: GW_138\"\t\"ID: GW_139\"\t\"ID: GW_140\"\t\"ID: GW_141\"\t\"ID: GW_142\"\t\"ID: GW_143\"\t\"ID: GW_144\"\t\"ID: GW_145\"\t\"ID: GW_146\"\t\"ID: GW_147\"\t\"ID: GW_148\"\t\"ID: GW_149\"\t\"ID: GW_150\"\t\"ID: GW_151\"\t\"ID: GW_152\"\t\"ID: GW_153\"\t\"ID: GW_154\"\t\"ID: GW_155\"\t\"ID: GW_156\"\t\"ID: GW_157\"\t\"ID: GW_158\"\t\"ID: GW_159\"\t\"ID: GW_160\"\t\"ID: GW_163\"\t\"ID: GW_164\"\t\"ID: GW_165\"\t\"ID: GW_166\"\t\"ID: GW_167\"\t\"ID: GW_168\"\t\"ID: GW_169\"\t\"ID: GW_170\"\t\"ID: GW_171\"\t\"ID: GW_172\"\t\"ID: GW_173\"\t\"ID: GW_174\"\t\"ID: GW_175\"\t\"ID: GW_177\"\t\"ID: GW_178\"\t\"ID: GW_179\"\t\"ID: GW_180\"\t\"ID: GW_181\"\t\"ID: GW_182\"\t\"ID: GW_183\"\t\"ID: GW_184\"\t\"ID: GW_185\"\t\"ID: GW_186\"\t\"ID: GW_187\"\t\"ID: GW_188\"\t\"ID: GW_189\"\t\"ID: GW_190\"\t\"ID: GW_191\"\t\"ID: GW_192\"\t\"ID: GW_193\"\t\"ID: GW_196\"\t\"ID: GW_197\"\t\"ID: GW_198\"\t\"ID: GW_199\"\t\"ID: GW_200\"\t\"ID: GW_201\"\t\"ID: GW_202\"\t\"ID: GW_203\"\t\"ID: GW_204\"\t\"ID: GW_205\"\t\"ID: GW_206\"\t\"ID: GW_207\"\t\"ID: GW_209\"\t\"ID: GW_210\"\t\"ID: GW_211\"\t\"ID: GW_212\"\t\"ID: GW_214\"\t\"ID: GW_215\"\t\"ID: GW_216\"\t\"ID: GW_217\"\t\"ID: GW_218\"\t\"ID: GW_219\"\t\"ID: GW_220\"\t\"ID: GW_221\"\t\"ID: GW_222\"\t\"ID: GW_223\"\t\"ID: GW_224\"\t\"ID: GW_225\"\t\"ID: GW_226\"\t\"ID: GW_227\"\t\"ID: GW_228\"\t\"ID: GW_229\"\t\"ID: GW_230\"\t\"ID: GW_231\"\t\"ID: GW_233\"\t\"ID: GW_234\"\t\"ID: GW_235\"\t\"ID: GW_236\"\t\"ID: GW_237\"\t\"ID: GW_238\"\t\"ID: GW_239\"\t\"ID: GW_240\"\t\"ID: GW_241\"\t\"ID: GW_242\"\t\"ID: GW_243\"\t\"ID: GW_244\"\t\"ID: GW_245\"\t\"ID: GW_246\"\t\"ID: GW_247\"\t\"ID: GW_248\"\t\"ID: GW_249\"\t\"ID: GW_250\"\t\"ID: GW_251\"\t\"ID: GW_253\"\t\"ID: GW_255\"\t\"ID: GW_256\"\t\"ID: GW_257\"\t\"ID: GW_258\"\t\"ID: GW_262\"\t\"ID: GW_263\"\t\"ID: GW_265\"\t\"ID: GW_266\"\t\"ID: GW_268\"\t\"ID: GW_269\"\t\"ID: GW_271\"\t\"ID: GW_272\"\t\"ID: GW_275\"\t\"ID: GW_280\"\t\"ID: GW_281\"\t\"ID: GW_282\"\t\"ID: GW_283\"\t\"ID: GW_284\"\t\"ID: GW_285\"\t\"ID: GW_286\"\t\"ID: GW_287\"\t\"ID: GW_288\"\t\"ID: GW_289\"\t\"ID: GW_290\"\t\"ID: GW_292\"\t\"ID: GW_293\"\t\"ID: GW_294\"\t\"ID: GW_295\"\t\"ID: GW_296\"\t\"ID: GW_297\"\t\"ID: GW_298\"\t\"ID: GW_299\"\t\"ID: GW_300\"\t\"ID: GW_301\"\t\"ID: GW_302\"\t\"ID: GW_303\"\t\"ID: GW_305\"\t\"ID: GW_306\"\t\"ID: GW_307\"\n",
      "\n",
      "`!Sample_characteristics_ch1` 행을 찾았습니다.\n",
      "파일에서 추출된 모든 샘플의 총 개수: 270개\n",
      "\n",
      "오류: 'normal' 샘플을 찾을 수 없습니다.\n"
     ]
    }
   ],
   "source": [
    "# 필요한 라이브러리를 불러옵니다.\n",
    "import os\n",
    "\n",
    "# 데이터 파일 경로를 지정합니다.\n",
    "file_path = os.path.join(\"..\",\"data\", \"GSE65858_series_matrix.txt\")\n",
    "\n",
    "print(f\"파일 경로: {file_path}\\n\")\n",
    "\n",
    "try:\n",
    "    with open(file_path, 'r', encoding='utf-8') as f:\n",
    "        # 이 변수는 !Sample_characteristics_ch1 행에서 추출한 그룹 정보를 저장할 것입니다.\n",
    "        group_info = []\n",
    "        found_line = False\n",
    "\n",
    "        # 파일의 모든 행을 순회합니다.\n",
    "        for line in f:\n",
    "            # '!Sample_characteristics_ch1' 행을 찾습니다.\n",
    "            if \"!Sample_characteristics_ch1\" in line:\n",
    "                print(line)\n",
    "                print(\"`!Sample_characteristics_ch1` 행을 찾았습니다.\")\n",
    "                found_line = True\n",
    "                \n",
    "                # 탭으로 구분된 값을 리스트로 변환합니다.\n",
    "                # strip() 함수를 사용해 양쪽 공백을 제거하고,\n",
    "                # strip('\"')을 사용해 큰따옴표를 제거합니다.\n",
    "                group_info = [x.strip().strip('\"') for x in line.split('\\t')]\n",
    "                \n",
    "                # 첫 번째 항목인 '!Sample_characteristics_ch1'은 샘플 그룹이 아니므로 제거합니다.\n",
    "                group_info.pop(0)\n",
    "\n",
    "                # 추출한 샘플 그룹 정보를 출력합니다.\n",
    "                print(f\"파일에서 추출된 모든 샘플의 총 개수: {len(group_info)}개\")\n",
    "                \n",
    "                # 'normal'과 'carcinoma' 샘플을 분리합니다.\n",
    "                # 이 행은 'tissue: normal' 또는 'tissue: carcinoma'와 같은 형식일 수 있습니다.\n",
    "                normal_samples = [group for group in group_info if 'normal' in group.lower()]\n",
    "                carcinoma_samples = [group for group in group_info if 'carcinoma' in group.lower()]\n",
    "                \n",
    "                if normal_samples:\n",
    "                    print(f\"\\n성공: 총 {len(normal_samples)}개의 'normal' 샘플을 찾았습니다.\")\n",
    "                    print(f\"성공: 총 {len(carcinoma_samples)}개의 'carcinoma' 샘플을 찾았습니다.\")\n",
    "                    \n",
    "                    print(\"\\n정상 샘플 예시:\")\n",
    "                    for sample in normal_samples[:5]:\n",
    "                        print(f\" - {sample}\")\n",
    "                        \n",
    "                    print(\"\\n암 샘플 예시:\")\n",
    "                    for sample in carcinoma_samples[:5]:\n",
    "                        print(f\" - {sample}\")\n",
    "                else:\n",
    "                    print(\"\\n오류: 'normal' 샘플을 찾을 수 없습니다.\")\n",
    "                \n",
    "                # 목적을 달성했으므로 반복문을 종료합니다.\n",
    "                break\n",
    "        \n",
    "        if not found_line:\n",
    "            print(\"오류: `!Sample_characteristics_ch1` 행을 찾지 못했습니다.\")\n",
    "\n",
    "except FileNotFoundError:\n",
    "    print(f\"오류: `{file_path}` 파일을 찾을 수 없습니다. 파일 경로를 다시 확인해 주세요.\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6a4790ed",
   "metadata": {},
   "source": [
    "파일의 모든 메타데이터 행이 `!`로 시작하고, 이 행들만 무시하고 넘어가면 데이터 행을 바로 읽을 수 있음을 확인."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "8c690028",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "파일을 성공적으로 불러왔습니다.\n",
      "         ID_REF  GSM1607684  GSM1607685  GSM1607686  GSM1607687  GSM1607688  \\\n",
      "0  ILMN_1343291     13.1886     13.4787     13.2263     13.4400     13.2678   \n",
      "1  ILMN_1343295     11.5546     11.1959     11.0929     11.7418     10.4042   \n",
      "2  ILMN_1651209      6.4138      6.4093      6.3680      6.3737      6.4413   \n",
      "3  ILMN_1651221      6.4520      6.4427      6.4834      6.4753      6.4617   \n",
      "4  ILMN_1651228     12.4126     12.3789     12.5750     12.4906     12.0646   \n",
      "\n",
      "   GSM1607689  GSM1607690  GSM1607691  GSM1607692  ...  GSM1607944  \\\n",
      "0     13.4199     13.3665     13.7561     13.3718  ...     13.5946   \n",
      "1     11.7005     11.1597     11.4082     11.9006  ...     10.9917   \n",
      "2      6.4673      6.5391      6.3631      6.4349  ...      6.4887   \n",
      "3      6.5220      6.4703      6.4411      6.5209  ...      6.4846   \n",
      "4     12.4749     12.2534     12.1751     12.0844  ...     12.3579   \n",
      "\n",
      "   GSM1607945  GSM1607946  GSM1607947  GSM1607948  GSM1607949  GSM1607950  \\\n",
      "0     13.3038     13.4352     13.1511     13.5872     13.4713     13.5859   \n",
      "1     11.4681     11.6038     10.2602     11.7133     11.6153     10.6864   \n",
      "2      6.3490      6.5698      6.4373      6.4324      6.3617      6.3804   \n",
      "3      6.6523      6.4805      6.4332      6.3026      6.5231      6.7199   \n",
      "4     12.3669     12.4104     12.1868     12.1160     12.4291     12.1365   \n",
      "\n",
      "   GSM1607951  GSM1607952  GSM1607953  \n",
      "0     13.4941     13.4830     13.4524  \n",
      "1     11.1780     11.6972     11.0559  \n",
      "2      6.4352      6.4303      6.4314  \n",
      "3      6.4903      6.3808      6.5302  \n",
      "4     12.1809     11.8767     12.0883  \n",
      "\n",
      "[5 rows x 271 columns]\n",
      "\n",
      "--- 데이터프레임 정보 ---\n",
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 31330 entries, 0 to 31329\n",
      "Columns: 271 entries, ID_REF to GSM1607953\n",
      "dtypes: float64(270), object(1)\n",
      "memory usage: 64.8+ MB\n",
      "\n",
      "--- 데이터프레임 통계 요약 ---\n",
      "         GSM1607684    GSM1607685    GSM1607686    GSM1607687    GSM1607688  \\\n",
      "count  31330.000000  31330.000000  31330.000000  31330.000000  31330.000000   \n",
      "mean       7.336770      7.332219      7.329406      7.338121      7.337224   \n",
      "std        1.256594      1.246241      1.251399      1.256685      1.252411   \n",
      "min        5.761700      6.008100      6.033000      6.075500      6.081800   \n",
      "25%        6.472800      6.475000      6.475400      6.475300      6.476500   \n",
      "50%        6.762000      6.758400      6.757250      6.763150      6.766100   \n",
      "75%        7.813500      7.811975      7.791575      7.809800      7.815275   \n",
      "max       13.416000     13.510500     13.632800     13.578500     13.633500   \n",
      "\n",
      "         GSM1607689    GSM1607690    GSM1607691    GSM1607692    GSM1607693  \\\n",
      "count  31330.000000  31330.000000  31330.000000  31330.000000  31330.000000   \n",
      "mean       7.338554      7.339351      7.334711      7.339488      7.345404   \n",
      "std        1.260875      1.258564      1.256302      1.266818      1.264828   \n",
      "min        6.033600      6.077800      6.049300      6.072300      5.823000   \n",
      "25%        6.470500      6.470700      6.471900      6.465500      6.471400   \n",
      "50%        6.767900      6.767350      6.766050      6.770450      6.772400   \n",
      "75%        7.801675      7.821675      7.800600      7.797775      7.816900   \n",
      "max       14.014000     13.436800     13.756100     13.961200     13.500800   \n",
      "\n",
      "       ...    GSM1607944    GSM1607945    GSM1607946    GSM1607947  \\\n",
      "count  ...  31330.000000  31330.000000  31330.000000  31330.000000   \n",
      "mean   ...      7.337137      7.331814      7.336286      7.332370   \n",
      "std    ...      1.255937      1.260957      1.259298      1.247696   \n",
      "min    ...      6.138100      6.047800      6.143000      6.167900   \n",
      "25%    ...      6.477700      6.463400      6.469600      6.486100   \n",
      "50%    ...      6.762100      6.765600      6.774750      6.746250   \n",
      "75%    ...      7.815325      7.783000      7.791400      7.806550   \n",
      "max    ...     13.932000     13.514200     13.480900     13.292700   \n",
      "\n",
      "         GSM1607948    GSM1607949    GSM1607950    GSM1607951    GSM1607952  \\\n",
      "count  31330.000000  31330.000000  31330.000000  31330.000000  31330.000000   \n",
      "mean       7.330844      7.334991      7.332052      7.330264      7.329684   \n",
      "std        1.249971      1.265942      1.262019      1.258020      1.245088   \n",
      "min        6.117100      6.138000      6.135400      6.069700      6.162100   \n",
      "25%        6.469100      6.469500      6.464600      6.465125      6.470200   \n",
      "50%        6.757800      6.751550      6.768350      6.746800      6.768950   \n",
      "75%        7.818675      7.812600      7.788375      7.823750      7.789600   \n",
      "max       13.587200     13.844000     13.611300     13.666800     13.492200   \n",
      "\n",
      "        GSM1607953  \n",
      "count  31330.00000  \n",
      "mean       7.33567  \n",
      "std        1.26368  \n",
      "min        6.15850  \n",
      "25%        6.46850  \n",
      "50%        6.76115  \n",
      "75%        7.80480  \n",
      "max       13.69150  \n",
      "\n",
      "[8 rows x 270 columns]\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "\n",
    "# 파일 경로를 지정\n",
    "file_path = os.path.join(\"..\", \"data\", \"GSE65858_series_matrix.txt\")\n",
    "\n",
    "try:\n",
    "    # '!'로 시작하는 모든 주석 줄을 건너뛰고 데이터를 불러옴.\n",
    "    df = pd.read_csv(file_path, sep='\\t', comment='!', encoding='utf-8')\n",
    "    print(\"파일을 성공적으로 불러왔습니다.\")\n",
    "    print(df.head())\n",
    "    # 데이터 정보 확인\n",
    "    print(\"\\n--- 데이터프레임 정보 ---\")\n",
    "    df.info()\n",
    "\n",
    "    # 데이터 통계량 요약\n",
    "    print(\"\\n--- 데이터프레임 통계 요약 ---\")\n",
    "    print(df.describe())\n",
    "\n",
    "except FileNotFoundError:\n",
    "    print(f\"오류: '{file_path}' 경로에 파일이 존재하지 않습니다. 경로를 다시 확인해주세요.\")\n",
    "except Exception as e:\n",
    "    print(f\"알 수 없는 오류가 발생했습니다: {e}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2446a078",
   "metadata": {},
   "source": [
    "이제 이 코드를 실행하면 Pandas가 데이터를 성공적으로 불러오는 것을 확인할 수 있음.\n",
    "\n",
    "\n",
    "## 데이터 전처리 및 가공\n",
    "\n",
    "통계 분석에 바로 사용할 수 있도록 데이터를 깔끔하게 정리 가공해야함. 현재 데이터는 유전자 발현량과 샘플 정보가 뒤섞여 있어서 그대로 사용하기 어렵기 때문에 다음과 같은 작업을 진행.\n",
    "1. 샘플 정보 추출: `!Sample_title`이라는 열에서 '두경부암(cancer)'과 '건강한 샘플(control)' 정보를 추출해 별도의 컬럼으로 만들고, 해당 컬럼을 **group**으로 명명.\n",
    "2. 통계 분석이나 머신러닝을 할 때 사용하는 scikit-learn이나 statsmodels 같은 파이썬 라이브러리들은 **'행이 샘플, 열이 특징'**인 구조의 데이터를 기대하지만, 해당 데이터는 행이 유전자, 열이 샘플인 형태. 때문에 전치가 필요.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0c7c8f08",
   "metadata": {},
   "source": [
    "1단계: 원본 데이터 (Raw Data)\n",
    "| ID_REF | GSM1578505 | GSM1578506 | ... |\n",
    "| :--- | :--- | :--- | :--- |\n",
    "| !Sample_title | head and neck cancer | normal control | ... |\n",
    "| ILMN_1311 | 12.5 | 13.1 | ... |\n",
    "| ILMN_1524 | 10.2 | 11.5 | ... |\n",
    "\n",
    "대략 이런형태를 가지고 있고, ID_REF는 유전자의 고유한 시별자, 이름표 개념이다.\n",
    "ILMN은 Illumina의 약자로 유전자 분석하는 칩을 만드는 유명한 회사로, 해당 회사의 칩에 있는 ㅇ전자들은 ILMN이라는 고유한 번호를 사용. GEO는 Gene Expression Omnibus의 약자로 NCBI가 운영하는 공개 유전체 데이터베이스.\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.13.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
